build/azDevOps/azure/api-contract-tests.yml

```diff
@@ -57,6 +57,14 @@ stages:
           vmImage: 'ubuntu-latest'
         continueOnError: False
         steps:
+
+          - task: UseDotNet@2
+            displayName: 'Use .NET Core SDK 6.0.x'
+            inputs:
+              packageType: sdk
+              version: "6.0.x"
```

Comment:
Isn't there a variable for the version?

Reply:
Contract test code hasn't changed, there's no variable for this currently.

Reply:
Ahh Contract tests in general aren't thought through well in Stacks. Noöne knows the ideal PACT flow and it's been poorly implemented currently.